Dockerfile

```diff
@@ -7,10 +7,10 @@ COPY environment.yml /
 RUN conda env create --quiet -f /environment.yml && conda clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
-ENV PATH /opt/conda/envs/nf-core-eager-2.2.2dev/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-eager-2.2.2/bin:$PATH
 
 # Dump the details of the installed packages to a file for posterity
-RUN conda env export --name nf-core-eager-2.2.2dev > nf-core-eager-2.2.2dev.yml
+RUN conda env export --name nf-core-eager-2.2.2 > nf-core-eager-2.2.2.yml
```

Comment:
These changes are because you didn't bump the version number in `nextflow.config` on `dev`:
https://github.com/nf-core/eager/blob/0d7131f35e475ccba46c9c6cf15686e627b45151/nextflow.config#L340